gremlin-server/src/main/java/org/apache/tinkerpop/gremlin/server/Settings.java

```diff
@@ -387,6 +387,18 @@ public SerializerSettings() {}
         public String className = AllowAllAuthenticator.class.getName();
 
         /**
+         * Enable audit logging of authenticated users and gremlin evaluation requests.
+         */
+        public boolean enableAuditLog = false;
```

Comment:
What is the purpose of this configuration option? It doesn't appear to be used in any way.